CHANGELOG.md

```diff
@@ -8,13 +8,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Add Time To Live (TTL) for all tables.
 - DB migration for creating new `housekeeping_metadata` table.
+- Integration test for tables with TTL.
```

Comment:
Since most end users don't really care about or use the tests we generally don't mention changes to them in the CHANGELOG.

Reply:
Ping ^